### PR TITLE
Fix `pg list` table alignment

### LINKED ISSUE #1628: CLI shows timeline ids instead of branch names
The `zenith pg list` command displays timeline IDs instead of branch names:
```
/git-sandbox/neon (avoid-rel-extend-in-wal-redo)$ ZENITH_REPO_DIR=/data/zenith-data/ ./target/release/zenith pg list
NODE	ADDRESS	TIMELINE	BRANCH NAME	LSN		STATUS
main	127.0.0.1:55432	b3b135512e1e1e2af08113c792424e80	main	0/16CA810	running
```
Let's map the timeline ID to human-readable branch name, if the mapping exists.

### PR DESCRIPTION
**Pull request by aome510**
Fixes #1628

- add [`comfy_table`](https://github.com/Nukesor/comfy-table/tree/main) and use it to construct table for `pg list` CLI command

## Comparison

- Old:

```
NODE	ADDRESS	TIMELINE	BRANCH NAME	LSN		STATUS
main	127.0.0.1:55432	3823dd05e35d71f6ccf33049de366d70	main	0/16FB140	running
migration_check	127.0.0.1:55433	3823dd05e35d71f6ccf33049de366d70	main	0/16FB140	running
```

- New:

```
 NODE             ADDRESS          TIMELINE                          BRANCH NAME  LSN        STATUS  
 main             127.0.0.1:55432  3823dd05e35d71f6ccf33049de366d70  main         0/16FB140  running 
 migration_check  127.0.0.1:55433  3823dd05e35d71f6ccf33049de366d70  main         0/16FB140  running 
```

### CODE REVIEW

**Comment by hlinnaka** [comment]
> Matter of taste, but I think I'd prefer to not have the borders. Like docker for example:

Can comfy-table do that easily?

File: zenith/src/main.rs [hunk 1]
```diff
@@ -665,7 +665,17 @@ fn handle_pg(pg_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
 
             let timeline_name_mappings = env.timeline_name_mappings();
 
-            println!("NODE\tADDRESS\tTIMELINE\tBRANCH NAME\tLSN\t\tSTATUS");
+            let mut table = comfy_table::Table::new();
+
+            table.set_header(vec![
```
Comment (SomeoneToIgnore) [inline zenith/src/main.rs:670]:
```suggestion
            table.set_header(&[
```

File: zenith/src/main.rs [hunk 2]
```diff
@@ -684,16 +694,17 @@ fn handle_pg(pg_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
                     .map(|name| name.as_str())
                     .unwrap_or("?");
 
-                println!(
-                    "{}\t{}\t{}\t{}\t{}\t{}",
-                    node_name,
-                    node.address,
-                    node.timeline_id,
+                table.add_row(vec![
```
Comment (SomeoneToIgnore) [inline zenith/src/main.rs:697]:
```suggestion
                table.add_row(&[
```

**Review by SomeoneToIgnore — APPROVED** [review]
A few styling nits, LGTM overall

**Pull request merged**